src/commands/lambda/instrument.ts

```diff
@@ -444,6 +447,8 @@ export class InstrumentCommand extends Command {
       environment,
       extensionVersion,
       extraTags,
+      fips: this.fips || this.fipsConfig.fips,
```

Comment:
If possible, let's also remove `fips` from the `InstrumentationSettings`, since I believe it's used nowhere anyway.